interfaces/Portalicious/src/app/pages/project-payments/components/create-payment/create-payment.component.ts

```diff
@@ -186,9 +201,22 @@ export class CreatePaymentComponent {
     this.createPaymentMutation.mutate({
       dryRun: this.currentStep() === 1,
       paginateQuery: actionData.query,
+      paymentId: this.nextPaymentId(),
     });
   }
 
+  hasIntegratedFsp = computed(() =>
+    this.dryRunResult()?.fspsInPayment.some(
+      (fsp) => fsp !== FinancialServiceProviders.excel,
+    ),
+  );
+
+  hasExcelFsp = computed(() =>
+    this.dryRunResult()?.fspsInPayment.includes(
+      FinancialServiceProviders.excel,
+    ),
+  );
```

Comment:
Isn't this the same result as the helper: `projectHasFspWithExportFileIntegration`?

And/or could this check live 'next to' that helper?

Reply:
> Isn't this the same result as the helper: projectHasFspWithExportFileIntegration?

Not quite, because `projectHasFspWithExportFileIntegration` could be true, but the `FspWithExportFileIntegration` might not have been used in this specific payment.

> And/or could this check live 'next to' that helper?

Will do

Reply:
Ah. yeah. I get that the specifics are different; But I meant it more like "they have a similar purpose"..

But now all the helpers can be refactored in the same place 🎉 .